dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -120,16 +120,16 @@ The file Makefile.PL should look something like this:
     WriteMakefile(
 	NAME         => 'Mytest',
 	VERSION_FROM => 'Mytest.pm', # finds $VERSION
-	LIBS         => [''],   # e.g., '-lm'
-	DEFINE       => '',     # e.g., '-DHAVE_SOMETHING'
-	INC          => '',     # e.g., '-I/usr/include/other'
+	LIBS         => [''], # e.g., '-lm'
+	DEFINE       => '', # e.g., '-DHAVE_SOMETHING'
+	INC          => '-I.', # e.g., '-I. -I/usr/include/other'
     );
 
 The file Mytest.pm should start with something like this:
 
     package Mytest;
 
-    use 5.008008;
+    use 5.017007;
```

Comment:
I suggest we make this 5.020000. I don't think it's very likely that someone will use 5.17.1 to 5.17.6 when trying out the XS tutorial.

Objections?

Reply:
> I suggest we make this 5.020000.

I concur.

Reply:
The use perl version statement should accurately reflect the minimum perl version required to make the code run correctly.

Reply:
https://www.cpan.org/src/README.html

```
Major   Version   Released
5.20    5.20.0    2014-05-27
5.18    5.18.0    2013-05-18
```

<hr>

Copied from the replaced PR.

The define `PERL_NO_GET_CONTEXT` was added to `h2xs.PL`on Dec 6, 2012. 

The define `PERL_NO_GET_CONTEXT` was added to ParseXS's pods on Feb 16, 2014.

Perl `5.008008`  could have been updated to `5.017007` concomitant with either of those commits.

I used 5.017007 in the .pod believing PERL_NO_GET_CONTEXT first appeared in that release.

However it would be unusual to expose the reader to a development release of perl in the .pod, and maybe using instead the >5.017007 next stable perl version would be more sensible.

Reply:
`PERL_NO_GET_CONTEXT` provides an optimization. If it is used on earlier perl versions, it just has no impact. There is no reason to change the required version due to it.

Reply:
EXAMPLE 1

The reader is instructed to run "`h2xs -A -n Mytest`".

A rendering is presented of that command's generated Mytest.xs.

The rendering contains `#define PERL_NO_GET_CONTEXT`.

Factually if the define first appeared in 5.17.7 then having `5.008008` in the rendered Mytest.pm would not be consistent with what the reader would see in the command's generated file.

Reply:
The prose clearly says: (emphasis mine)

> Finally, the Mytest.xs file should look **something like this**:

There is no requirement for non-function-specific details to be identical, and this being a tutorial it would in fact even be better to: Explain in prose that `PERL_NO_GET_CONTEXT` is an optimization which can show up but is not required for function; and leave the version requirement low.